jupyter_core/troubleshoot.py

```diff
@@ -24,7 +24,7 @@ def subs(cmd):
     try:
         stdout = subprocess.check_output(cmd)
         return stdout.decode('utf-8').strip()
```

Comment:
While we're making this more robust, let's use the 'replace' error handler for decoding, so it won't crash if it gets non-UTF-8 output.